lib/components/term.js

```diff
@@ -131,11 +131,21 @@ export default class Term extends PureComponent {
     }
   }
 
-  onMouseUp() {
+  onMouseUp(e) {
     // handle copying selected text
     if (this.props.copyOnSelect && this.term.hasSelection()) {
       clipboard.writeText(this.term.getSelection());
     }
+
+    // handle quickEdit copy/paste with right click
+    if (this.props.quickEdit && e.button === 2) {
+      if (this.term.hasSelection()) {
+        clipboard.writeText(this.term.getSelection());
```

Comment:
Won't this be run twice? if you have both `quickEdit` and `copyOnSelect` activated?
can't we just change the if here https://github.com/zeit/hyper/pull/2392/files#diff-268710614c2274f611be667a5f72bb13R136

```
if ((this.props.copyOnSelect || this.props.quickEdit && e.button === 2) && this.term.hasSelection()) {
```
Otherwise it will be copied twice which could be a problem
🤔

Reply:
Ah yeah, if you select text _and then_ right click, the right click would copy the same text twice. It still works but you're right, unnecessary.

That condition you suggested isn't enough, because in the quickEdit case, it needs to clear the selection after copying. I modified it to just do one or the other, thoughts?

Reply:
Looks good to me! @derrickpelletier 👍